core/commands/custom.py

```diff
@@ -27,8 +27,17 @@ class GsCustomCommand(WindowCommand, GitCommand):
     """
 
     def run(self, **kwargs):
+        if '{PROMPT_ARG}' in kwargs.get('args'):
```

Comment:
Line 39/48 below (`if not args:`) should probably be moved up here, if you're going to assume the presence of an `args` keyword argument.  Otherwise, this line will throw an exception.